src/wmtk/operations/utils/UpdateEdgeOperationMultiMeshMapFunctor.cpp

```diff
@@ -156,7 +155,13 @@ void UpdateEdgeOperationMultiMeshMapFunctor::update_ear_replacement(
                     continue;
                 }
 
-                const int64_t parent_old_vid = m.id_vertex(parent_tuple);
+
+                parent_tuple = m.parent_scope(
```

Comment:
add A TODO / comment as to why this is necessary. this existing is a bug i think right? the parent_tuple should already be valid in the parent scope

Reply:
I also think parent_tuple should be already valid in the parent scope. But there is a case I met in tet, where if I don't resurrect it in the parent  scope,  it fails is_valid check in the parent scope. need to look into it later